src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -504,6 +504,17 @@ internal static OpenFileCommand DeserializeCore(XmlElement element)
             protected override void ExecuteCore(DynamoModel dynamoModel)
             {
                 dynamoModel.OpenFileImpl(this);
+
+                // Log file open action and the number of nodes in the opened workspace
+                Dynamo.Logging.Analytics.CreateFileOperationEvent(
+                    XmlFilePath, Logging.Actions.Open, dynamoModel.CurrentWorkspace.Nodes.Count());
+
+                // Log each unresolved node name in the opened workspace
+                foreach (DummyNode n in dynamoModel.CurrentWorkspace.Nodes.Where(x => x is DummyNode))
+                {
+                    Dynamo.Logging.Analytics.TrackEvent(Logging.Actions.Open, Logging.Categories.NodeOperations,
```

Comment:
Can we track this in one event, with a new Action `Unresolved`, value as number of unresolved, Description as comma separated string?